packages/teleport/src/Console/DocumentNodes/useNodes.ts

```diff
@@ -18,17 +18,22 @@ import { useEffect, useState } from 'react';
 import { useLocation } from 'react-router';
 import { FetchStatus, SortType } from 'design/DataTable/types';
 import useAttempt from 'shared/hooks/useAttemptNext';
+
 import history from 'teleport/services/history';
-import type { Node, NodesResponse } from 'teleport/services/nodes';
+
 import getResourceUrlQueryParams, {
   ResourceUrlQueryParams,
 } from 'teleport/getUrlQueryParams';
-import { useConsoleContext } from './../consoleContextProvider';
-import * as stores from './../stores';
+
 import labelClick from 'teleport/labelClick';
 import { AgentLabel } from 'teleport/services/agents';
 import { sortLogins } from 'teleport/Nodes/useNodes';
 
+import * as stores from './../stores';
+import { useConsoleContext } from './../consoleContextProvider';
+
+import type { Node, NodesResponse } from 'teleport/services/nodes';
+
```

Comment:
couple things wrong with `teleport/*` imports

- some are grouped eg line 28-30
- other has new lines
- one is out of order line 35

Reply:
These are all correct as per the config.